boards/common/arduino-atmega/include/board_common.h

```diff
@@ -76,9 +76,9 @@ extern "C" {
 #define LED2_ON             (PORTD &= ~LED2_MASK)
 #define LED2_TOGGLE         (PORTD ^=  LED2_MASK)
 #else
-#define LED0_ON             (PORTD |=  LED0_MASK)
-#define LED0_OFF            (PORTD &= ~LED0_MASK)
-#define LED0_TOGGLE         (PORTD ^=  LED0_MASK)
+#define LED0_ON             (PORTB |=  LED0_MASK)
```

Comment:
Could you instead move thos to the `#ifdef ATMEGA328P` ... `#endif` part at the top of this file? So that the `#else` part is dropped?

I don't think more AVR based arduinos will be released, but who knows.

Reply:
I moved this code to `#ifdef ATMEGA328P` and `#ifdef ATMEGA2560` sections. There is some code repetitions - but I think this is clearer than previously in `#else' section.